liberty/Liberty.i

```diff
@@ -265,9 +265,15 @@ default_operating_conditions()
 
 %extend LibertyCell {
 const char *name() { return self->name(); }
+bool has_timing_model() { return self->hasTimingArcs(); }
 bool is_leaf() { return self->isLeaf(); }
 bool is_buffer() { return self->isBuffer(); }
 bool is_inverter() { return self->isInverter(); }
+bool is_clock_gate() { return self->isClockGate(); }
+bool is_integrated_clock_gating_cell() { return self->isClockGate(); }
+bool is_memory() { return self->isMemory(); }
+bool is_sequential() { return self->hasSequentials(); }
+bool dont_use() { return self->dontUse(); }
```

Comment:
why do you need these AND properties?
I have migrated toward using get_property over swig object syntax.